docs/API-AND-PERMISSIONS.md

```
@@ -0,0 +1,470 @@
+    userId: ctx.session.user.discordUserId,
+  });
+  
+  throw error;
+}
+```
+
+### 6. Keep Routers Organized
+
+Group related procedures into routers by domain:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)